.github/workflows/build.yaml

```diff
@@ -15,12 +15,15 @@ jobs:
   builds:
     if: ${{ !(github.ref == 'refs/heads/main' && contains(github.event.head_commit.message, '[maven-release-plugin]')) }}
     needs: [tests]
+
     runs-on: ubuntu-latest
     steps:
-      - uses: browser-actions/setup-firefox@latest
-      - run: firefox --version
-      - uses: browser-actions/setup-geckodriver@latest
-      - run: geckodriver --version  
+      - uses: browser-actions/setup-chrome@latest
```

Comment:
https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#browsers-and-drivers

is there a reason the buildin chromedriver was not enough?